packages/dd-trace/src/proxy.js

```diff
@@ -180,6 +181,7 @@ class Tracer extends NoopProxy {
       if (!this._tracingInitialized) {
         this._tracer = new DatadogTracer(config)
         this.appsec = new AppsecSdk(this._tracer, config)
+        appsecStandalone.configure(config)
```

Comment:
maybe we could move this configure() somewhere in appsec territory ? it's not super important but if we can do it in a clean way it could be nice

Reply:
I need it there to replace the default `PrioritySampler` via `DatadogTracer` constructor...